src/eslint/help.js

```diff
@@ -23,7 +23,7 @@ function parseAlias(arr){
 
 function parseRegular(arr){
   if(!arr[0]){
-    return;
+    return false;
```

Comment:
It looks like for some reason, returning `undefined` here causes an early termination of the `_.each` loop in `parseHelp`. I have no idea why, but returning `false` here seems to fix it.